### PR TITLE
Simulation: more access to camera parameters

### PR DESCRIPTION
**Pull request by damienjadeduff**
This gives the user of the simulation module the ability to retrieve camera parameters and also to set the clipping planes, which earlier were set to arbitrary values.
We are using the simulation module in its current state and we expect others are too. 
Kind regards.

### CODE REVIEW

File: simulation/include/pcl/simulation/range_likelihood.h [hunk 1]
```diff
@@ -145,6 +164,18 @@ namespace pcl
         const float*
         getScoreBuffer ();
 
+        float 
+        getZNear(){ return z_near_; }
```
Comment (stefanbuettner) [inline simulation/include/pcl/simulation/range_likelihood.h]: Space before opening parenthesis.

File: simulation/include/pcl/simulation/range_likelihood.h [hunk 2]
```diff
@@ -99,6 +99,25 @@ namespace pcl
...
         /**
+         * Get the basic camera intrinsic parameters
+         */
+        void
+        getCameraIntrinsicsParameters (int &camera_width_in,
+                                      int &camera_height_in,
+                                      float &camera_fx_in,
+                                      float &camera_fy_in,
+                                      float &camera_cx_in,
+                                      float &camera_cy_in)
```
Comment (stefanbuettner) [inline simulation/include/pcl/simulation/range_likelihood.h]: Indentation: Missing space.

**Comment by stefanbuettner** [comment]
Hey damienjadeduff,

firstly, I think that's a good approach you're following there :+1:
Looks good to me, if you fix some minor coding style issues.

Cheers,
Stefan

**Comment by SergioRAgostinho** [comment]
Please squash the commits into a single one.

**Comment by damienjadeduff** [comment]
I made the requested white-space changes and squashed the commits. I made an assumption about the 2nd request and changed the pre-existing set() function in the same way. Regards.

**Comment by stefanbuettner** [comment]
Looks good to me

**Pull request merged**